src/components/RegistrationForm.js

```diff
@@ -0,0 +1,162 @@
+
+                    <div className={'form-group' + (submitted && !user.email ? ' has-error' : '')}>
+                        <label htmlFor="email">Your Email</label>
+                        <input onChange={this.handleInputChange} type="email" className="form-control"
+                            name="email" id="email" placeholder="Enter your Email" />
+                        {submitted && !user.email &&
+                            <div className="help-block">Email is required</div>
+                        }
+
+                    </div>
+
+                    <div className={'form-group' + (submitted && !user.password ? ' has-error' : '')}>
+                        <label htmlFor="password">Password</label>
+                        <input onChange={this.handleInputChange} type="password" className="form-control"
+                            name="password" id="password" placeholder="Enter your Password" />
+                        {submitted && !user.password &&
+                            <div className="help-block">Password is required</div>
+                        }
+                    </div>
+
+                    <div className={'form-group' + (submitted && (!user.confirmedPassword || !isSamePasswords) ? ' has-error' : '')}>
+                        <label htmlFor="confirm">Confirm Password</label>
+                        <input onChange={this.handleInputChange} type="password"
+                            className="form-control" name="confirmedPassword" id="confirm" placeholder="Confirm your Password" />
+                        {submitted && !user.confirmedPassword &&
+                            <div className="help-block">Verification is required</div>
+                        }
+                        {!isSamePasswords && submitted &&
+                            <div className="help-block">Passwords are not the same</div>
+                        }
+                    </div>
+
+                    <div className='form-group' >
+                        <label htmlFor="birthday">Birthday</label>
+                        <input onChange={this.handleInputChange} type="date" id="birthday" name="birthday" className="form-control"
+                            min="1950-01-01" max="2015-12-31" />
+                    </div>
+
+
+                    <div className="form-group">
+                        <div className="cols-sm-10">
+                            <pre className="input-group">
+                                <input onChange={this.handleInputChange} type="checkbox" id="isEmployee" name="isEmployee" />
+                                <label htmlFor="isEmployee" className="cols-sm-2 control-label" value="false"> Are you our employee?</label>
+                            </pre>
+                        </div>
+                    </div>
+
+                    <div className="form-group">
+                        {!registering && <button type="submit" className="btn btn-primary">Register</button>}
```

Comment:
I really don't like "registering" name of variable. If its a bool variable, please, change it.

Reply:
Done